cordova-lib/src/cordova/plugin.js

```diff
@@ -114,6 +116,22 @@ module.exports = function plugin(command, targets, opts) {
                             target = target.substring(0, target.length - 1);
                         }
 
+                        var parts = target.split('@');
+                        var id = parts[0];
+                        var version = parts[1];
+
+                        // If no version is specified, retrieve the version from config.xml
+                        if(!version && !cordova_util.isUrl(id) && !cordova_util.isDirectory(id)){
+                            events.emit('verbose', 'no version specified, retrieving version from config.xml');
+                            var ver = getVersionFromConfigFile(id, cfg);
+
+                            if( cordova_util.isUrl(ver) || cordova_util.isDirectory(ver) ){
```

Comment:
isDirectory don't check for null/undefined. Should either add that to isDirectory, or add check here.